lib/mongo/collection.rb

```diff
@@ -184,11 +190,11 @@ def create
       if (options[:collation] || options[Operation::COLLATION]) && !server.features.collation_enabled?
         raise Error::UnsupportedCollation.new
       end
-      Operation::Commands::Create.new({
-                                        selector: operation,
-                                        db_name: database.name,
-                                        write_concern: write_concern
-                                      }).execute(server)
+      Operation::Commands::Create.new(
```

Comment:
Are operations like creating and deleting collection supposed to also be done in the context of the session? I can't find anything specific in the spec about them, so I'm not sure.